master/pkg/tasks/task.go

```diff
@@ -44,7 +44,7 @@ func workDirArchive(aug *model.AgentUserGroup) container.RunArchive {
 // non-root container features will not work properly.
 func injectUserArchive(aug *model.AgentUserGroup) container.RunArchive {
 	passwdBytes := []byte(
-		fmt.Sprintf("%v:x:%v:%v::%v:/bin/sh\n", aug.User, aug.UID, aug.GID, ContainerWorkDir),
+		fmt.Sprintf("%v:x:%v:%v::%v:/bin/bash\n", aug.User, aug.UID, aug.GID, ContainerWorkDir),
```

Comment:
wait, is _this_ why vscode shell was failing?  How on earth did you figure that out?

Reply:
yes. vscode plugin logs had `-sh: 4: function: not found` in them.